tools/compiletest/src/runtest.rs

```diff
@@ -452,6 +577,71 @@ impl<'test> TestCx<'test> {
         None
     }
 
+    // Search for the coverage related outputs
+    fn find_coverage_check_blocks(input: &str) -> Option<Vec<&str>> {
+        let mut blocks_with_unreachable_or_satisfied = Vec::new();
+
+        // Split the input text into blocks separated by empty lines
+        let blocks: Vec<&str> = input.split("\n\n").collect();
+
+        // Iterate through the blocks and find the ones containing "UNREACHABLE" or "SATISFIED"
+        for block in blocks {
+            if block.contains("UNREACHABLE") || block.contains("SATISFIED") {
+                blocks_with_unreachable_or_satisfied.push(block);
```

Comment:
Should be looking for `UNSATISFIED` and `SATISFIED` because those are the only two possible results for cover checks.